src/audioseal/models.py

```diff
@@ -73,13 +74,17 @@ def message(self, message: torch.Tensor) -> None:
     def get_watermark(
         self,
         x: torch.Tensor,
+        sample_rate: int,
         message: Optional[torch.Tensor] = None,
     ) -> torch.Tensor:
         """
         Get the watermark from an audio tensor and a message.
```

Comment:
Docstring update. Also as we have > 2 params now, can we update the docstring here in a proper format ?